rotkehlchen/tests/api/test_addressbook.py

```diff
@@ -604,19 +604,15 @@ def names_request(chain_addresses: list[OptionalChainAddress]) -> requests.Respo
                     name='Cody',
                     address=address_cody,
                     blockchain=SupportedBlockchain.ETHEREUM,
-                ), AddressbookEntry(
-                    name='rotki global db',
-                    address=address_rotki,
-                    blockchain=SupportedBlockchain.ETHEREUM,
-                ),
+                ),  # address_rotki is already added
```

Comment:
why this comment here? If it's a comment to the reviewer due to the diff then add a commen in the PR. Not in the code. Looking at it in the code it makes no sense. Remove it.

Also where was it already added? What change lead to this?